wiki/Article_styling_criteria/Writing/de.md

```diff
@@ -230,23 +239,98 @@ Manche Wörter haben Varianten. Ihre bevorzugte Schreibweise ist wie folgt:
 - `erstellt` anstatt `beatmapped` oder `mapped`.
 - `BN` oder `Beatmap Nominators`, wenn auf die *Beatmap Nominators* verwiesen wird.
 
-### Wettbewerbsartikel
+### Turnierartikel
 
 Diese Worte müssen wie folgt verwendet werden (der Abstand muss übereinstimmen):
 
 - `NoMods`, `NoMod` oder `NM`
 - `FreeMods`, `FreeMod` oder `FM`
 - `Tiebreaker`
 
-## Abkürzungen, Akronyme und Initialwörter
-
-Abkürzungen, Akronyme und Initialwörter müssen bei ihrem ersten Auftreten vollständig ausgeschrieben werden. Nachfolgende Vorkommen sind optional, aber werden nur ausgeschrieben, sofern erforderlich. Das Folgende ist ein Beispiel:
+### Übersetzungen für häufige Begriffe
+
+Dieser Abschnitt listet häufig verwendete Begriffe und ihre entsprechenden Übersetzungen auf. Beim Übersetzen von Artikeln sollte darauf geachtet werden, möglichst die hier aufgelisteten Übersetzungen zu verwenden. Ausnahmen können gemacht werden, wenn die ursprüngliche Bedeutung durch die Übersetzung verloren geht.
+
+#### Gameplay
+
+| Englisch | Deutsch |
+| :-: | :-- |
+| `game modifier (mod)` | `Spielmodifikation (Mod)` |
+| `gameplay` | `Spielverlauf` |
+| `multiplier` | `Multiplikator` |
+| `pass rate` | `Erfolgsquote` |
+| `Combo` | `Combo` |
+| `beatmap difficulty` | `Schwierigkeitsgrad/-stufe der Beatmap` |
+| `beatmap category` | `Beatmap-Kategorie` |
+| `playcount` | `Spielanzahl` |
+| `ranked` | `ranked` oder `gerankt` |
+| `qualified` | `qualified` oder `qualifiziert` |
+| `loved` | `loved` oder `geliebt` |
+| `approved` | `approved` oder `bestätigt` |
+| `overall difficulty` | `Allgemeine Schwierigkeit` |
+| `medal` | `Achievement` oder `Medaille` |
+
+#### Spielelemente
+
+| Englisch | Deutsch |
+| :-: | :-- |
+| `fruits` | `Früchte` |
```

Comment:
I used `Fruits` in the [hit object article](https://osu.ppy.sh/wiki/de/Hit_object), because i saw other translations just used the english terms as well and its probably more common among players anyway.
```suggestion
| `fruits` | `Fruits` |
```